CONTRIBUTING.md

```diff
@@ -121,28 +121,13 @@ $ make test
    ...
    ```
 
-3. Build the image:
+3. Install Chaos Mesh:
 
-   ```bash
-   $ make image-chaos-mesh
-   $ make image-chaos-dahsboard
-   $ make image-chaos-daemon
-   # or build all images
-   $ make image
-   ```
-
-4. Load image into Kubernetes nodes:
-
-   ```bash
-   $ kind load docker-image pingcap/chaos-mesh:latest
-   $ kind load docker-image pingcap/chaos-dashboard:latest
-   $ kind load docker-image pingcap/chaos-daemon:latest
-   ```
-
-5. Deploy Chaos Mesh:
+   Following command will rebuild project code and reinstall chaos mesh.
 
    ```bash
-   $ ./install.sh --runtime containerd
+   #
+   $ ./hack/local-up-operator.sh
```

Comment:
How about renaming this file to `local-up-chaos-mesh.sh`?

Reply:
ok